docs/content/api.md

```diff
@@ -50,9 +50,9 @@ The animation will continue to run until the `mover` in the physics simulation h
 
 #### Example
 
-```typescript
-const GravityBasic: React.FC = () => {
-  const [props] = useGravity<HTMLDivElement>({
+```playground
```

Comment:
🎉

Reply:
: ( I also notice that changing the example causes the demo to stop running.